src/app/store/hooks/usePost.tsx

```diff
@@ -10,9 +10,11 @@ export const usePost = ( id: number ): Post => {
             rendered: 'Test content',
         },
         meta: {
-            _thumbnail_id: 2,
-        }
-    }
+            _thumbnail_id: 1,
```

Comment:
FYI Attachments and Posts share the same ID sequence, an attachment is actually a post object as well under the hood, shares the same DB table. Hence the `2`